timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimeLockServices.java

```diff
@@ -31,35 +32,31 @@ static TimeLockServices create(
             TimestampService timestampService,
             LockService lockService,
             AsyncTimelockService timelockService,
-            AsyncTimelockResource timelockResource,
-            TimestampManagementService timestampManagementService) {
+            TimestampManagementService timestampManagementService,
+            LockLog lockLog) {
         return ImmutableTimeLockServices.builder()
                 .timestampService(timestampService)
                 .lockService(lockService)
                 .timestampManagementService(timestampManagementService)
                 .timelockService(timelockService)
-                .timelockResource(timelockResource)
+                .lockLog(lockLog)
                 .build();
     }
 
     TimestampService getTimestampService();
 
     LockService getLockService();
-    // The Jersey endpoints
-    AsyncTimelockResource getTimelockResource();
     // The RPC-independent leadership-enabled implementation of the timelock service
     AsyncTimelockService getTimelockService();
 
     TimestampManagementService getTimestampManagementService();
 
+    // Do not use without atlasdb team guidance
```

Comment:
Perhaps use a javadoc so that it shows up in an IDE? Or mark as deprecated which seems to be what we do for telling people not to use things.

It's a bit painful that we're even exposing it tbh, but I don't have a nicer idea off the top of my head.

Reply:
I did both. Makes sense.